common/src/main/java/com/datastrato/gravitino/dto/file/FilesetDTO.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
```

Comment:
Can you please update the year to 2024 for newly added files in your PR.

Reply:
Done.